frontend/public/plugins.ts

```diff
@@ -22,10 +22,11 @@ const getI18nNamespaces = () => {
 };
 
 // The '@console/active-plugins' module is generated during a webpack build,
+// Console active plugins module has its source generated during webpack build,
 // so we use dynamic require() instead of the usual static import statement.
 const activePlugins =
   process.env.NODE_ENV !== 'test'
-    ? (require('@console/active-plugins').default as ActivePlugin[])
+    ? (require('../get-active-plugins').default as ActivePlugin[])
```

Comment:
Aha, this is an important change.

Before :arrow_forward: use `webpack-virtual-modules` plugin to add new virtual `@console/active-plugins` JS module to the webpack compilation and then pull in its source here via `require()` call.

After :arrow_forward: use `val-loader` to transform the source of existing `get-active-plugins` JS module and then pull in that source here via `require()` call.